### DISCUSSION TITLE
Adonis 4.1 Models

### DISCUSSION THREAD

**Discussion by Hiba-Ed** [adonisjs/core#1617] [Help] [answered]
Hello. I was going through the [documentation](https://adonisjs.com/docs/4.1/lucid) and found this:

createdAtColumn
The field name used to set the creation timestamp (return null to disable):

updatedAtColumn
The field name used to set the modified timestamp (return null to disable):

Does this mean that we can return null from the methods above to disable Adonis from setting the creation and modified timestamp? Here is what my migration looks like:

```
      this.create('employee_info', (table) => {
        table.increments('id')
        table.string('name', 100)
        table.string('address', 255)
      })
```

**Comment by thetutlage** [comment, marked as answer]
Yes. Returning `null` will tell the model that you are not using `created_at` and `updated_at` timestamps

**Reply by Hiba-Ed** [reply]
Thank you.
